switzerland/src/main/java/org/eqasim/switzerland/scenario/SwissConfigAdapter.java

```diff
@@ -2,24 +2,34 @@
 
 import org.matsim.core.config.*;
 
+import java.util.Arrays;
+import java.util.List;
 import java.util.function.Consumer;
 
 
 public class SwissConfigAdapter {
-    static public void run(String[] args, ConfigGroup[] modules, Consumer<Config> adapter)
+    protected static Boolean hasCustomActivities = false;
+    protected static List<String> activityTypes;
+
+    public static void run(String[] args, ConfigGroup[] modules, Consumer<Config> adapter)
             throws CommandLine.ConfigurationException {
         CommandLine cmd = new CommandLine.Builder(args) //
                 .requireOptions("input-path", "output-path") //
-                .allowOptions("activity_list") //
+                .allowOptions("activity-list") //
                 .build();
 
-        if(cmd.hasOption("activity_list")) {
-            CustomActivities.run(cmd.getOption("activity_list").get());
+        if(cmd.hasOption("activity-list")) {
+            setCustomActivities(cmd.getOption("activity-list").get());
         }
 
         Config config = ConfigUtils.loadConfig(cmd.getOptionStrict("input-path"), modules);
         adapter.accept(config);
 
         new ConfigWriter(config).write(cmd.getOptionStrict("output-path"));
     }
-}
+
+    public static void setCustomActivities (String activityList) {
```

Comment:
Why is this method public and static? Is it used anywhere outside this class?

Reply:
It is static because the class calling it are static. I would change the access modifier to protected

Reply:
Excellent, thanks. Then it is good to go